scss/_patterns_table-sortable.scss

```diff
@@ -39,7 +39,7 @@
     }
 
     &[aria-sort]:hover {
-      color: $color-link;
+      color: $colors--theme--link;
```

Comment:
You may need to rebase this now that links are merged.

We did have link variables doubled, so this one is gone, and the current one is called `$colors--theme--link-default`